src/lib/converter/plugins/GitHubPlugin.ts

```diff
@@ -82,9 +90,13 @@ class Repository {
             });
         }
 
-        out = <ShellJS.ExecOutputReturnValue> ShellJS.exec('git rev-parse --short HEAD', {silent: true});
-        if (out.code === 0) {
-            this.branch = out.stdout.replace('\n', '');
+        if (!this.revision) {
+            out = <ShellJS.ExecOutputReturnValue> ShellJS.exec('git rev-parse --short HEAD', {silent: true});
+            if (out.code === 0) {
+                this.branch = out.stdout.replace('\n', '');
+            }
+        } else if (this.revision) {
+            this.branch = this.revision;
```

Comment:
Is there a better name we can think of than revision? What's the git correct name?

Also, what if we just made `this.branch = revision` and only run the `if` instead of requiring an `if else`? Do we need both pieces of information?